deploy/scripts/src/mobile/testSteps/backend.ts

```diff
@@ -52,13 +40,7 @@ export function getBiometricTokenV2(sessionId: string): void {
       authSessionId: sessionId
     })
 
-    timeRequest(
-      () =>
-        http.get(biometricTokenUrl, {
-          tags: { name: 'GET /biometricToken/v2' }
```

Comment:
Likely a bit behind the times - what's the rationale for removing the tags? Truthfully I don't know what value it provided beforehand...

Reply:
The value is that it provides a `name` tag on all metrics, although in these implementations the value is exactly the same as the `group` tag value which is already being tagged on all metrics. So removing it removes a bit of unnecessary duplication.

Also as an aside in other scripts we removed the `name` tag as it usually defaults to the URL of the API call which can, in cases where this includes a large JWT query parameter or some other large unique ID, cause very large metric payload sizes putting a strain on the OpenTelemetry pipeline

Reply:
nice thank yous